spec.html

```diff
@@ -1942,7 +1942,7 @@ <h4>Layout of operators</h4>
                         <code>T<sub>ascent</sub></code> +
                         <code>T<sub>descent</sub></code>
                         and at position determined by the previous box metrics
-                        shifted by Δ towards the <a>line-over</a>.
+                        shifted by Δ towards the <a>line-under</a>.
```

Comment:
I woud need to think about this direction / sign error, but can it be a separate issue / PR? if it's just a mistake we don't need the WG to approve a behavior change